docs/dotnet/dotnet-aspnetcore-metrics.md

```diff
@@ -0,0 +1,208 @@
+# Semantic Conventions for ASP.NET Core 8.0 metrics
```

Comment:
How are these conventions intended to be evolved? Do we expect separate "Semantic Conventions for ASP.NET Core 9.0 metrics" in the future?

Reply:
Moved version to notes on the individual instruments, removed from doc title